packages/nextjs/pages/index.tsx

```diff
@@ -179,23 +202,27 @@ const Home: NextPage = () => {
 
       {/* Block Explorer */}
       <div className="bg-base-200">
-        <div className="container max-w-[90%] lg:max-w-7xl m-auto py-24 lg:py-20 lg:pl-12 lg:pr-6 lg:pt-28 flex flex-col-reverse lg:flex-row items-center gap-5 lg:gap-0">
-          <div className="space-y-6">
-            <div className="flex items-center justify-center lg:flex-col lg:items-start lg:justify-start gap-2 pt-4 lg:pt-0">
+        <div className="container max-w-[90%] lg:max-w-7xl m-auto py-24 lg:pb-20 lg:pt-28 lg:pl-4  flex flex-col justify-between items-center lg:flex-row lg:items-start gap-5 lg:gap-0">
+          <div className="flex flex-col items-center">
+            <div className="max-w-[400px] lg:max-w-none">
+              <Image src="/assets/se2BlockExplorer.png" alt="Block Explorer" width={1400} height={1400} />
+            </div>
+          </div>
+
+          <div className="space-y-6 flex flex-col items-center lg:pl-16 lg:pt-4">
```

Comment:
First time I have opened this last version, the "*Block Explorer*" section felt a bit too close to "*Buidl in Community*" band.

Checking old version with previous zig-zag, it seemed that the text content was a bit farther from the band, maybe we could add a bit more of `pt` on `lg` devices, at least to that text content? Maybe just a personal perception tbh 😅.

Previous:

![image](https://github.com/scaffold-eth/scaffoldeth.io/assets/55535804/1ec4904f-5bc8-4106-9dbc-a82f5f574f89)